src/main/kotlin/step3/racingcar/domain/CurrentTurn.kt

```diff
@@ -0,0 +1,20 @@
+package step3.racingcar.domain
+
+class CurrentTurn(var _currentTurn: Int = 0) {
+    val value: Int
+        get() = _currentTurn
+
+    init {
+        require(
+            START_TURN <= _currentTurn
+        )
```

Comment:
```suggestion
        require(START_TURN <= _currentTurn)
```
한 줄로 처리가 가능하면 한 줄로 해주시는 것이 좋을 것 같아요

Reply:
과한 라인 변경 이었군요..